bot.user.js

```diff
@@ -572,10 +571,11 @@ window.computeFoodGoal = function() {
                     clusterSize += 1;
                 }
             }
-            clusterScore /= p1.distance;
+            clusterAbsScore = clusterScore;
+            clusterScore /= p1.distance * Math.sqrt(p1.distance);
```

Comment:
I see, the penalty is the distance to the 1.5 power. You could also do `/= Math.pow(p1.distance, 1.5);`